client/src/services/upload-thumbnail.ts

```diff
@@ -0,0 +1,17 @@
+import firebase from "./firebase";
+
+const storage = firebase.storage();
+const storageRef = storage.ref("thumbnail_images");
+
+ const upload = async (file: File | null, id: string):Promise<string> => {
```

Comment:
nit: `?File` is equivalent to `File | null`
```
const upload = async (file: ?File, id: string):Promise<string> => {
```

Reply:
I kept getting `JSDoc types can only be used inside documentation comments.ts(8020)` after changing it to `?File`. Do you know what this means?